resources/database/database_reset_rm.sql

```diff
@@ -26,7 +26,7 @@ ALTER SEQUENCE collectionexercise.sampleunitgrouppkseq RESTART WITH 1;
 ALTER SEQUENCE collectionexercise.sampleunitpkseq RESTART WITH 1;
 ALTER SEQUENCE collectionexercise.samplelinkpkseq RESTART WITH 1;
 
-UPDATE collectionexercise.collectionexercise SET statefk = 'INIT';
```

Comment:
Adding 
UPDATE collectionexercise.collectionexercise SET statefk = 'SCHEDULED' where statefk = 'READY_FOR_REVIEW';
here will fix the problem @benjefferies mentioned about not being able to run the tests multiple times.